README.md

```diff
@@ -16,72 +20,166 @@ make
 ## Upload to a container registry
 
 e.g.
-
 ```bash
 docker push quay.io/kiegroup/kie-cloud-operator:<version>
 ```
 
-## Deploy to OpenShift 4.5+ using OLM
+### Note
+If your images pushed to quay.io may are set as private you need to that user needs to have the quay.io pull secret set in OCP 
+otherwise you need to set all operator related images as public. 
+If this is not set then image pulls will be failing.
+
+## Deploy to OpenShift 4.7+ using OLM
 
-To install this operator on OpenShift 4 for end-to-end testing, make sure you have access to a quay.io account to create
-an application repository. Follow the [authentication](https://github.com/operator-framework/operator-courier/#authentication) 
-instructions for Operator Courier to obtain an account token. 
+To install this operator on OpenShift 4 for end-to-end testing, make sure you have access to a quay.io (https://quay.io/) account to create
+an application repository. Follow the [authentication](https://github.com/operator-framework/operator-courier/#authentication)
+instructions for Operator Courier to obtain an account token.
 This token is in the form of "basic XXXXXXXXX" and both words are required for the command.
 
-If pushing to another quay repository, replace _kiegroup_ with your username or other namespace. 
-Also note that the push command does not overwrite an existing repository, 
-and it needs to be deleted before a new version can be built and uploaded. 
-Once the bundle has been uploaded, create an [Operator Source](https://github.com/operator-framework/community-operators/blob/master/docs/testing-operators.md#linking-the-quay-application-repository-to-your-openshift-40-cluster) 
+Also note that the push command does not overwrite an existing repository,
+and it needs to be deleted before a new version can be built and uploaded.
+Once the bundle has been uploaded, create an [Operator Source](https://github.com/operator-framework/community-operators/blob/master/docs/testing-operators.md#linking-the-quay-application-repository-to-your-openshift-40-cluster)
 to load your operator bundle in OpenShift.
 
-**Create your own index image**
+**Create your own Bundle**
 
-Requires [opm](https://github.com/operator-framework/operator-registry/releases) v1.15.3+ -
+i.e. 7.12.0-1 version
```

Comment:
improve this.

start with something like, 

To create the bundle image follow the steps ...